src/app/components/search-results/search-results.component.ts

```diff
@@ -50,12 +54,35 @@ export class SearchResultsComponent implements OnInit {
           dataTemplate: this.template
         }
       ],
-      showFooter: true
+      showFooter: true,
+      enableSort: true,
+      defaultSortColumn: 'name',
+      defaultSortDir: 'asc'
```

Comment:
The two defaults above are optional parameters. Right? Where you set their default values?

Reply:
Those should not be in the config. I've now put them in a separate object passed trough an Input to the component.
They denote by which column and direction is the table currently sorted. This usually comes from the service response.